MobiusCore/Source/MobiusController.swift

```diff
@@ -48,21 +46,20 @@ public final class MobiusController<Model, Event, Effect> {
         viewQueue: DispatchQueue
     ) {
         let actualLoopQueue = DispatchQueue(label: "MobiusController \(Model.self)", target: loopQueue)
-        modelToStartFrom = initialModel
-        self.loopQueue = actualLoopQueue
         self.viewQueue = viewQueue
 
-        var running: () -> Bool = { false }
+        let state = State(model: initialModel, queue: actualLoopQueue)
```

Comment:
Capturing `self` in `flipEventsToLoopQueue` is impossible because of definite initialization requirements, but I’ll add a comment to clarify the overall structure here